src/Xamarin.Android.Tools.AndroidSdk/Sdks/AndroidSdkWindows.cs

```diff
@@ -163,6 +162,37 @@ private static IEnumerable<string> GetOpenJdkPaths ()
 			}
 		}
 
+		/// <summary>
+		/// Locate OpenJDK installations by well known path.
+		/// </summary>
+		/// <returns>List of valid OpenJDK paths in version descending order.</returns>
+		private static IEnumerable<string> GetKnownOpenJdkPaths ()
```

Comment:
I did the same as in XamarinVS, https://github.com/xamarin/XamarinVS/blob/82fd693d01115cd38ccdf1d8cdc28b4a0ee65dda/src/Core/VisualStudio.Android/Services/Setup/JavaSdkSetup.cs#L294